firmware/lib/Comm/src/ArduinoSerialDevice.h

```diff
@@ -15,7 +15,7 @@ class ArduinoSerialDevice: public SerialDevice
         bool ser_connect(uint32_t baud_rate);
```

Comment:
Same as in another PR - maybe remove `ser_` from class methods

Reply:
See reply in the other PR